03-seeking-help.Rmd

```diff
@@ -124,6 +124,14 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
```

Comment:
Good point, maybe adding a tip about tsv beeing a variant of csv and looking into csv help with help.